src/EditorFeatures/CSharpTest/ImplementAbstractClass/ImplementAbstractClassTests.cs

```diff
@@ -32,6 +32,27 @@ internal override (DiagnosticAnalyzer, CodeFixProvider) CreateDiagnosticProvider
                  SingleOption(CSharpCodeStyleOptions.PreferExpressionBodiedProperties, CSharpCodeStyleOptions.NeverWithSilentEnforcement),
                  SingleOption(CSharpCodeStyleOptions.PreferExpressionBodiedIndexers, CSharpCodeStyleOptions.NeverWithSilentEnforcement));
 
+        internal Task TestAllOptionsOff73Async(
```

Comment:
Ah, did I miss deleting this function when I did my undo? Want to undo it further?

Reply:
Sure.